src/lib/y2partitioner/actions/edit_blk_device.rb

```diff
@@ -93,13 +93,38 @@ def title
       #
       # @return [Boolean] true if the edit action can be performed; false otherwise.
       def run?
-        error = extended_partition_error || lvm_thin_pool_error
-        return true unless error
+        return true if errors.empty?
+
+        # Only first error is shown
+        Yast::Popup.Error(errors.first)
 
-        Yast::Popup.Error(error)
         false
       end
 
+      # Errors when trying to edit a device
+      #
+      # @return [Array<Strings>]
+      def errors
+        [used_device_error, extended_partition_error, lvm_thin_pool_error].compact
+      end
+
+      # Error when trying to edit an used device
+      #
+      # @note A device is being used when it forms part of an LVM or MD RAID.
+      #
+      # @return [String, nil] nil if the device is not being used.
+      def used_device_error
+        return nil unless device.part_of_lvm_or_md?
+
+        # TRANSLATORS: Error message when trying to edit an used device. %{name} is
+        # replaced by the device name (e.g., /dev/sda1).
```

Comment:
Not sure if this line will be added to the .pot file. Please place it in the line below, right before the call to `_()`

Reply:
Not to mention than I don't think our translators need a note explaining them than the string "The device is in use. It cannot be used"  is an error about the user trying to edit an used device. :wink: